src/block/Anvil.php

```diff
@@ -94,4 +101,26 @@ public function place(BlockTransaction $tx, Item $item, Block $blockReplace, Blo
 	public function tickFalling() : ?Block{
 		return null;
 	}
+
+	public function onHitGround(FallingBlock $blockEntity) : void{
+		if($this->position->isValid()){
+			foreach($this->position->getWorld()->getCollidingEntities($blockEntity->getBoundingBox()) as $ent){
+				if($ent instanceof Living){
+					if($blockEntity->fallDistance > 1){
+						//If player has helmet do 1/4 the normal damage
```

Comment:
The damage is reduced **by** 1/4, not **to** 1/4 (at least, according to the wiki).

Reply:
Yeah I tested it you are correct.